docs/core/deploying/deploy-with-cli.md

```diff
@@ -1,201 +1,154 @@
+Your app is configured to target a specific version of .NET Core, and that runtime is required on the machine where you want to run your app. For example, if your app targets .NET Core 2.2, any machine that your app runs on must have the .NET Core 2.2 runtime installed. As stated in the [Publishing basics](#publishing-basics) section, you can edit your project file to change the default target framework or to target more than one framework.
 
-Deploy the published files in any way you like. For example, you can package them in a Zip file, use a simple `copy` command, or deploy them with any installation package of your choice.
+You must (except for .NET Core 3.x) use the following switches with the `dotnet publish` command to publish an FDE:
 
-The following is the complete *csproj* file for this project.
+- `-r <RID>`  
+  This switch is used to specify which platform to compile an executable for and uses a runtime identifier (RID). For a list of runtime identifiers, see [Runtime Identifier (RID) catalog](..\rid-catalog.md). The target runtime identifier must support the selected version of .NET Core. 
 
-```xml
-<Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
-    <RuntimeIdentifiers>win10-x64;osx.10.11-x64</RuntimeIdentifiers>
-  </PropertyGroup>
-</Project>
-```
+- `--self-contained false`  
+  This switch is used to tell the .NET Core SDK to create the executable (that targets the RID) as an FDE.
 
-## Self-contained deployment with third-party dependencies
+Whenever you use the `-r` switch, the output folder path changes to: `<CURRENT-FOLDER>/bin/<BUILD-CONFIGURATION>/<TFM>/<RID>/publish/`
 
-Deploying a self-contained deployment with one or more third-party dependencies involves adding the dependencies. Two additional steps are required before you can run the `dotnet restore` ([see note](#dotnet-restore-note)) command:
+If you were to use the [example app](#sample-app), run `dotnet publish -f netcoreapp2.2 -r win10-x64 --self-contained false`. This creates the following executable: `<CURRENT-FOLDER>/bin/Debug/netcoreapp2.2/win10-x64/publish/apptest1.exe`
 
-1. Add references to any third-party libraries to the `<ItemGroup>` section of your *csproj* file. The following `<ItemGroup>` section uses Json.NET as a third-party library.
+> [!Note]
+> You can reduce the total size of your deployment by enabling **globalization invariant mode**. This mode is useful for applications that are not globally aware and that can use the formatting conventions, casing conventions, and string comparison and sort order of the [invariant culture](xref:System.Globalization.CultureInfo.InvariantCulture). For more information about **globalization invariant mode** and how to enable it, see [.NET Core Globalization Invariant Mode](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/globalization-invariant-mode.md)
```

Comment:
I think that globalization invariant mode only affects self-contained deployments. Is that correct, @tarekgh?

Reply:
Not really. any deployment can turn the invariant mode at anytime. We just turn it on in some deployments (e.g. Alpine) and it is turned off on the other deployments.